### ISSUE TITLE
Building emgu cv on linux arm64

### ISSUE THREAD

**Issue by Dzirit** [emgucv/emgucv#387]
Hi! 
Do you have any .sh scripts for configure cmake and build emgucv for Ubuntu 18.04 arm64? I need run my .net core app on this platform.
Maybe anybody knows how right configure cmake for linux arm64?

**Comment by emgucv** [comment]
You can try this here:
https://github.com/emgucv/emgucv/tree/master/platforms/ubuntu/20.04 
The scripts are used for ubutnu 20.04 x64, it may need some small changes to work on Ubuntu 18.05 arm64.

**Comment by Dzirit** [comment]
Could you help me? I have error on  one of steps building:

> -- CSC_EXECUTABLE_120: /usr/bin/csc
> -- MSBUILD_EXECUTABLE_120 : /usr/bin/msbuild
> -- CSC_EXECUTABLE_140: /usr/bin/csc
> -- MSBUILD_EXECUTABLE_140 : /usr/bin/msbuild
> -- CSC_EXECUTABLE_150: /usr/bin/csc
> -- MSBUILD_EXECUTABLE_150 : /usr/bin/msbuild
> -- CSC_EXECUTABLE_160: /usr/bin/csc
> -- MSBUILD_EXECUTABLE_160 : /usr/bin/msbuild
> -- Found csc: /usr/bin/csc
> -- Found gacutil: /usr/bin/gacutil
> -- Found al: /usr/bin/al
> -- Found resgen: /usr/bin/resgen
> -- Found msbuild: /usr/bin/msbuild
> -- Could not find vstool
> -- Could not find dotnet
> CMake Error at CMakeLists.txt:269 (MESSAGE): dotnet core is required to build Emgu CV on Ubuntu, please follow the
>   instructions to install dotnet core on Ubuntu:
>   https://dotnet.microsoft.com/download/linux-package-manager/ubuntu18-04/sdk-cu                                                                                                                                                             rrent
> 
> -- Configuring incomplete, errors occurred!

But I have installed  dotnet.  
dotnet --info:

> Runtime Environment:
>  OS Name:     ubuntu
>  OS Version:  18.04
>  OS Platform: Linux
>  RID:         ubuntu.18.04-arm64
>  Base Path:   /home/my/dotnet/sdk/3.1.404/
> 
> Host (useful for support):
>   Version: 3.1.10
>   Commit:  1721e39439
> 
> .NET Core SDKs installed:
>   3.1.404 [/home/my/dotnet/sdk]
> 
> .NET Core runtimes installed:
>   Microsoft.AspNetCore.App 3.1.10 [/home/my/dotnet/shared/Microsoft.AspNetCore.App]
>   Microsoft.NETCore.App 3.1.10 [/home/my/dotnet/shared/Microsoft.NETCore.App]

I don't know what to do next..

**Comment by emgucv** [comment]
If you install dotnet after calling CMake configuration on Emgu CV, you may need to clean up the repository and re-call the cmake config script. CMake may have cached the dotnet find_program result.

If you start from a fresh checkout, with dotnet already installed, cmake should find the dotnet command. 

Btw, in the master branch, Emgu CV will need dotnet 5.0 instead of 3.1 to compiled the demo projects. Some of our demo projects are now based on dotnet 5.0. If you want to use dotnet 3.1 instead of dotnet 5, you should checkout the 4.4 tag instead.

**Comment by Dzirit** [comment]
I installed dotnet manually to %HOME path. And add dotnet to enviroment path. I did everything according to the instructions https://docs.microsoft.com/ru-ru/dotnet/core/install/linux-ubuntu#manual-install. 
I checked your cmake file and found that you cmake looking for dotnet at /usr/local/share/dotnet.
I copied my dotnet folder from %HOME to /usr/local/share and it worked! I successfuly run  my face detection .net app  on linux arm64 platform. 
Many thanks for support!
